server/config/config.js

```diff
@@ -28,8 +28,20 @@ const creds = {
   dialectOptions,
 };
 
+const [username, password, host, port, database] = env.DATABASE_URL.replace(/[/:@]/g, ' ').split(' ').slice(3);
```

Comment:
this is neat, i dig it - you can use regex in `split` so you can probably just bypass the `replace` 👀 

once you make the change i'll attempt a prod deploy from thsi branch and we'll see how it goes!

Reply:
I've made this correction.